R/test.data.table.R

```diff
@@ -127,12 +149,19 @@ test <- function(num,x,y=TRUE,error=NULL,warning=NULL,output=NULL) {
     actual.err <<- conditionMessage(e)
     e
   }
+  if (memtest) {
+    timestamp = as.numeric(Sys.time())
+  }
   if (is.null(output)) {
     x = tryCatch(withCallingHandlers(x, warning=wHandler), error=eHandler)
     # save the overhead of capture.output() since there are a lot of tests, often called in loops
   } else {
     out = capture.output(print(x <<- tryCatch(withCallingHandlers(x, warning=wHandler), error=eHandler)))
   }
+  if (memtest) {
+    mem = as.data.frame(as.list(c(inittime=inittime, timestamp=timestamp, test=num, ps_mem(), gc_mem())))
+    fwrite(mem, "memtest.csv", append=TRUE)
```

Comment:
`fwrite` accepts a raw `list()`. If the items are unnamed, no column names are written.
```R
> fwrite(list(1,2,3))
1,2,3
> fwrite(list(foo=1,2,3))
foo,"",""
1,2,3
```
So, iiuc, it can be just : 
```R
mem = list(inittime, timestamp, num, ps_mem(), gc_mem())
fwrite(mem, "memtest.csv", append=TRUE)
```